src/autocomplete/postgresql/grammar/PostgreSqlParser.g4

```diff
@@ -402,7 +402,6 @@ alterTableStatement
     | ALTER TABLE ALL IN_P TABLESPACE name (OWNED BY roleList)? SET TABLESPACE name optionalNowait
     | ALTER INDEX (IF_P EXISTS)? indexName (alterTableCommands | indexPartitionCommand)
     | ALTER INDEX ALL IN_P TABLESPACE name (OWNED BY roleList)? SET TABLESPACE name optionalNowait
-    | ALTER SEQUENCE (IF_P EXISTS)? qualifiedName alterTableCommands
```

Comment:
This rule is very weird and doesn't work. You can basically write `ALTER SEQUENCE test_sequence ADD COLUMN`, which is incorrect.